### PR TITLE
[stable/nextcloud] Fix doc for default persistance value

### PR DESCRIPTION
**Pull request by pfuhrmann**
#### What this PR does / why we need it:
Fixing incorrect docs

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped

### CODE REVIEW

File: stable/nextcloud/README.md [hunk 1]
```diff
@@ -116,4 +116,4 @@ $ helm install --name my-release -f values.yaml stable/nextcloud
 The [Nextcloud](https://hub.docker.com/_/nextcloud/) image stores the nextcloud data and configurations at the `/var/www/html` paths of the container.
 
 Persistent Volume Claims are used to keep the data across deployments. This is known to work in GCE, AWS, and minikube.
-See the [Configuration](#configuration) section to configure the PVC or to disable persistence.
\ No newline at end of file
+See the [Configuration](#configuration) section to configure the PVC or to disable persistence.
```
Comment (chrisingenhaag) [inline stable/nextcloud/README.md]: Thx for the patch. Didn´t you mean 'to enable persistence' here? And I think we need a blank line at the end.

Comment (pfuhrmann) [reply stable/nextcloud/README.md]: Good catch. Reworded a bit. There is also a blank line on the end :)

**Comment by pfuhrmann** [comment]
@scottrigby can you have a look here, please?

**Pull request merged**